app/src/main/res/layout-sw600dp-land/onboarding_fragment.xml

```diff
@@ -54,6 +54,8 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="32dp"
         android:layout_marginBottom="16dp"
+        android:accessibilityTraversalAfter="@id/onboarding_fragment_next_image_view"
+        android:contentDescription="@{@string/onboarding_slide_dots_content_description(viewModel.slideNumber + 1)}"
```

Comment:
The correct format would be to use `String.format(` similar to `story_chapter_view` -> `chapter_title`

Reply:
Done.